### ISSUE TITLE
Upsetplot with vennpie: upsetplot legend colors don't correspond with vennpie legend

### ISSUE THREAD

**Issue by djakubosky** [GuangchuangYu/ChIPseeker#32]
Hi, 
Just wanted to add an issue here, could be that I made some mistake, but when I use the upsetplot with the vennpie=TRUE, the vennpie colors do not match up with the correct upsetplot colors on the set size axis.  Thanks for designing a very nice tool package! 

![image](https://cloud.githubusercontent.com/assets/15019958/14835496/e25481e0-0bbe-11e6-8f42-0b6cedb71cda.png)

Cheers, 
David

**Comment by GuangchuangYu** [comment]
do you have any recommendation, since not all information was in the vennpie, for example 5'UTR and 3'UTR were not shown and only upstream promoter was shown.

It's actually difficult to make the color consistent. Currently, you can pass `sets.bar.color="black"` to the `upsetplot` function to not coloring the set bar.

**Comment by djakubosky** [comment]
Thats a good point, I had simply noticed that in your vignette, the colors of matching features of the vennpie that did exist matched in color with the set size bar chart.  I'm not familiar enough with the upsetplot implementation in R to suggest a solution, but I would imagine that it might be possible to use a dictionary to set the specific color for any given reported feature, settling on a specific set of colors for any given "standardly" labeled feature of vennpie and upsetplots, however this is not extensible to custom annotations.  Tricky problem.    

Another caveat with my example is that I used your tool to annotate **copy number variants** which of course is not what you designed this tool for, but I believe this visualization type is great for, since we are often interested in what CNVs contain/overlap.  

Anyway thanks for the prompt reply, I'll just remove upsetplot coloring as suggested.
